lib/utils.js

```diff
@@ -658,6 +658,10 @@ exports.intersects = intersects;
  * @private
  */
 function interleave(arr, delimiter) {
+  if (!arr.length) {
```

Comment:
Oh, I should probably fix that in my code base 😅 
Also you should probably return `[]`.

Reply:
My bad